lighthouse-cli/chrome-launcher.js

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+const childProcess = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const inquirer = require('inquirer');
+const net = require('net');
+const rimraf = require('rimraf');
+
+const spawn = childProcess.spawn;
+const execSync = childProcess.execSync;
+
+const platformError = new Error('Platform ' + process.platform + ' is not supported');
+
+function defaults(val, def) {
+  return typeof val === 'undefined' ? def : val;
+}
+
+module.exports = class Launcher {
+  constructor(opts) {
+    opts = opts || {};
+    // choose the first one (default)
+    this.head = defaults(opts.head, true);
```

Comment:
I put an extra canary in Downloads so i could try this out.. 

ask() ends up with something like this:
![image](https://cloud.githubusercontent.com/assets/39191/18499473/1b1f8e5e-79f4-11e6-9fa1-f33180067a86.png)

(which is pretty cool)

but since `this.head` is set at the top, the 3rd case of the switch statement will always be set and I'm not clear on when ask() is called. (i commented out the 3rd case statement in order to get the above screenshot)

AFAIUI, `head` aka `select-chrome` mean, go with the first canary in your list of multiple, after you've sorted by priority. This basically means, "eh, if you find a few, just pick the one you like the most, lighthouse :)" given that, I think we'd want to tweak both names.